patientsearch/src/js/components/PatientListTable.js

```diff
@@ -150,6 +152,7 @@ export default function PatientListTable(props) {
   }
   const getPatientPMPSearchURL = (data) => {
     const dataURL = "/external_search/Patient";
+    if (data.id) return `${dataURL}/${data.id}`;
```

Comment:
@achen2401 we need to include `/external_search` ONLY when data.id is not present.  Actually, it's more complicated - we need to include /external_search anytime we don't have `data.id` or we don't have the identifier you use to paint some rows white.